.github/workflows/trivy.yml

```diff
@@ -30,3 +30,17 @@ jobs:
         uses: github/codeql-action/upload-sarif@v2
         with:
           sarif_file: 'trivy-results.sarif'
+      
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.24.0
+        with:
+          image-ref: 'docker.io/securefederatedai/openfl:${{ github.sha }}'
+          format: 'table'
+          exit-code: '1'
+          ignore-unfixed: true
+          vuln-type: 'os,library'
+          severity: 'CRITICAL,HIGH,MEDIUM,LOW'
+  
+      - name: Log completion message
+        run: |
+          echo "Trivy scan completed. Review the output above for vulnerabilities."
```

Comment:
Where are the trivy scan results published?

Reply:
in the above step it just echo the statements i.e echo "Trivy scan completed. Review the output above for vulnerabilities."

above this step we will Upload Trivy scan results to GitHub Security tab and also logs in the pipeline

![Screenshot 2024-09-25 at 5 19 43 PM](https://github.com/user-attachments/assets/cd661a36-d69d-4da2-a574-009f2d872c11)